keyboards/chocv/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2021 BrickBots Labs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// place overrides here
```

Comment:
Let's remove this file, it serves no purpose.

Reply:
Great!  It's gone 👍

Reply:
It's still here.

Reply:
Wow, all I can say is 'Whoops'.  Sorry about the mix up and time sink here.  Thanks both for you time.